cmd/notation/logout.go

```diff
@@ -16,8 +16,9 @@ func logoutCommand(opts *logoutOpts) *cobra.Command {
 		opts = &logoutOpts{}
 	}
 	return &cobra.Command{
-		Use:   "logout [server]",
+		Use:   "logout [flags] [server]",
```

Comment:
If server is an optional argument, https://github.com/notaryproject/notation/blob/f747031135a6010d92c737ea611276b18f4188d7/cmd/notation/logout.go#L21-L27 should we remove this logic?

Reply:
Currently, we only has a -h flag. If -h flag is specified, this logic will be skipped.